recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ outputs:
       host:
         - python
         - deprecated >=1.2.13
-        - numba >=0.53,<0.55
+        - numba >=0.53
```

Comment:
This is preventing me from installing `sktime` so good to see this will be fixed in the next release!👍

Reply:
no worries, I'll loosen up the restriction, but having some trouble resolving deps for this release.

Reply:
`sktime` seems to be working with number version relaxed